.travis.yml

```diff
@@ -1,15 +1,9 @@
 language: node_js
 node_js:
-  - "node"
-  - "lts/*"
-before_script:
-  - npm run build
-jobs:
-  - name: "src"
-    env: TEST_SUBJECT=src
-  - name: "lib"
-    env: TEST_SUBJECT=lib
-  - name: "dist"
-    env: TEST_SUBJECT=dist
-  - name: "e2e"
-    script: yarn run test:e2e
\ No newline at end of file
+  - 14
+  - 12
+  - 10
+before_install:
+  - npm i -g jest
```

Comment:
why do we need to install jest globally in travis before installing anything else?
should jest be in our package.json?

Reply:
it is but travis has some bug I wasn't able to solve where jest won't work without installing globally, calling it directly from node_modules also didn't solve it

Reply:
This sounds a bit strange.
we have multiple projects using jest with Travis, and none of them require this step.

Reply:
thanks fixed